JoinSEEDS/seeds_light_wallet
# P2P web view

### 🗃 Github Issue Or Explanation for this PR. (What is it supposed to do and Why is needed)

#1391 

### ✅ Checklist

- [x] Github issue details are up to date for people to QA.
- [x] I have tested all my changes.

### 🕵️‍♂️ Notes for Code Reviewer

Integration of the p2p web client in an embedded way in the LW. The p2p transactions are signed in the LW and the LW sends results back to the p2p.
 To test this branch:

1. Set `feature_flag_p2p` firebase flag true. 
4. Log in usign a seeds account.
5. Go to explore press p2p card, then press launch wallet, confirm and send.
6. You will see a succes dialog, close it, then you see a loading, finally a registration screen. 

Once you registrered will see the home screen like in the GIF flow below, If you don't want register just delete the cache.


### 🙈 Screenshots

<img src="https://user-images.githubusercontent.com/42857405/148672277-1af7de3f-82e6-418e-96bf-8158d6c2e271.gif" width="250">

### 👯‍♀️ Paired with

"nobody"
